starter_code/app.js

```diff
@@ -9,9 +9,25 @@ const punkAPI = new PunkAPIWrapper();
 app.set('view engine', 'hbs');
 app.set('views', __dirname + '/views');
 app.use(express.static(path.join(__dirname, 'public')));
+hbs.registerPartials( __dirname + "/views/partials/" );
 
 app.get('/', (req, res, next) => {
-  res.render('index');
+  res.render('index.hbs');
+});
+
+app.get( "/beers", (req, res, next ) => {
+  punkAPI.getBeers()
+  .then(beers => {
+    res.locals.beer = beers;
+    res.render( "beers.hbs" );
+  })
+  .catch(error => {
+    console.log(error)
+  })
+});
+
+app.get( "/random-beer", ( req, res, next ) => {
```

Comment:
You're missing the logic of fetching the random beer and passing it as data to the view. But I guess you knew that already and just didn't have time to complete it :)